pages/common/convert.md

```diff
@@ -16,13 +16,13 @@
 
 `convert {{path/to/input_image.png}} -resize 640x480 {{path/to/output_image.png}}`
 
-- Horizontally append images:
+- Scale an image to have a specified file size:
 
-`convert {{path/to/image1.png path/to/image2.png ...}} +append {{path/to/output_image.png}}`
+`convert {{path/to/input_image.png}} -define jpeg:extent=512kb {{path/to/output_image.jpg}}`
 
-- Vertically append images:
+- Vertically/Horizontally append images:
 
-`convert {{path/to/image1.png path/to/image2.png ...}} -append {{path/to/output_image.png}}`
+`convert {{path/to/image1.png path/to/image2.png ...}} -append/+append {{path/to/output_image.png}}`
```

Comment:
```suggestion
`convert {{path/to/image1.png path/to/image2.png ...}} {{-append|+append}} {{path/to/output_image.png}}`
```
This is the general placeholder format we use for highlighting multiple options in the same example.